src/commands/createNewProject/ProjectCreateStep/PythonProjectCreateStep.ts

```diff
@@ -30,10 +32,34 @@ const defaultRequirements: string = `# DO NOT include azure-functions-worker in
 azure-functions
 `;
 
+const azureWebJobsFeatureFlagsKey = 'AzureWebJobsFeatureFlags';
+
+const defaultFunctionAppSource = `import azure.functions as func
+
+app = func.FunctionApp(auth_level=func.AuthLevel.ANONYMOUS)
+
+# Uncomment to create an HTTP trigger-based function.
+# @app.function_name(name="HttpTrigger1")
+# @app.route(route="hello")
+# def test_function(req: func.HttpRequest) -> func.HttpResponse:
+#    return func.HttpResponse("HttpTrigger1 function processed a request!!!")
```

Comment:
```suggestion
#    return func.HttpResponse("HttpTrigger1 function processed a request!!!!!!!!!")
```

Reply:
I'm letting the Functions team decide how excited they want to be about a successfully processed request.